pulp_rpm/tests/functional/constants.py

```diff
@@ -609,13 +609,11 @@
 RPM_ONLY_METADATA_REPO_URL = urljoin(PULP_FIXTURES_BASE_URL, "rpm-unsigned-meta-only")
 
 AMAZON_MIRROR = "http://amazonlinux.us-east-1.amazonaws.com/2/core/latest/x86_64/mirror.list"
-CENTOS7_URL = "http://mirror.centos.org/centos-7/7/os/x86_64/"
-CENTOS7_OPSTOOLS_URL = "http://mirror.centos.org/centos/7/opstools/x86_64/"
 CENTOS8_STREAM_BASEOS_URL = "http://vault.centos.org/centos/8-stream/BaseOS/x86_64/os/"
 CENTOS8_STREAM_APPSTREAM_URL = "http://vault.centos.org/centos/8-stream/AppStream/x86_64/os/"
 CENTOS9_STREAM_BASEOS_URL = "http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/"
 CENTOS9_STREAM_APPSTREAM_URL = "http://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/"
-EPEL7_URL = "https://dl.fedoraproject.org/pub/epel/7/x86_64/"
+EPEL8_URL = "https://dl.fedoraproject.org/pub/epel/8/x86_64/"
```

Comment:
That link doesnt work, but that one does:
https://dl.fedoraproject.org/pub/epel/8/Everything/x86_64/

Reply:
Or https://dl.fedoraproject.org/pub/epel/8/Modular/x86_64/

Reply:
:+1:

Reply:
No, EPEL8 modular was shut down